config.yaml

```diff
@@ -4,6 +4,7 @@ tftp:
   port: 69
   retransmissions: 3
   maxParallelConnections: 10
+  filetransfertimeout: "60s"
```

Comment:
you sure you would like to introduce a transfer timeout on the server side? Or do I misunderstand this?